templates/pages/admin/form/access_control.html.twig

```diff
@@ -65,8 +65,6 @@
 
                 <section
                     class="mb-5"
-                    style="{{ access_control.fields.is_active == false ? "opacity: 0.5" : "" }}"
-                    {# Disabled item are showed with a lower opacity #}
                     data-glpi-toggle-control-target
                     data-glpi-access-control-form
```

Comment:
These are no longer used, I guess we can keep one of them on the new target node (see others suggestions)

```suggestion
```